.github/workflows/npm-package.yml

```diff
@@ -0,0 +1,21 @@
+name: Npm package
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        working-directory: typescript
+        run: |
+          npm install
```

Comment:
please use `npm ci`, npm install will regenerate the lock file, ci just install what is in the lock file

Reply:
Changed this to npm ci